app/controllers/users/profiles_controller.rb

```diff
@@ -5,17 +5,32 @@ class ProfilesController < ApplicationController
     before_action :authenticate_user!
     before_action :set_profile
     before_action :authorize_profile!
+    before_action :set_edit_facade, only: %i[edit update]
 
-    def index; end
+    def edit; end
+
+    def update
+      @facade = ::Profiles::EditFacade.new(@profile)
+      if ::Profiles::EditService.call(@profile, profile_params)
```

Comment:
right now you have @facade, so you should call everything there (have method where you call Profiles::EditService and check in controller if profile is saved)